docs/sphinx/examples/python/tutorials/quantum_fourier_transform.ipynb

```diff
@@ -251,6 +251,89 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Quantum Fourier Transform revisited\n",
+    "We provided one implementation of the quantum Fourier Transform above.  To demonstrate some of the other features of CUDA-Q, let's define a new kernel for the Quantum Fourier Transform, which we'll call `quantum_fourier_transform2`.  Pay attention to how the kernel definition differs from the example above.\n",
+    "\n",
+    " Since the quantum Fourier Transform is a unitary operator, its inverse is its adjoint.  Therefore, once we have implemented the quantum Fourier transform as a kernel in CUDA-Q which acts, we can use the built-in `adjoint` operation to create the inverse quantum Fourier transform."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
```

Comment:
The other cells' outputs in this notebook are already filled out (as if the notebook was already run). If you want the new cells to look the same in the published documentation, I think you need to run them locally and then commit the results into your PR. Is that what you, or do you want the published docs to leave these cells' output as empty?

Reply:
I'll run them and recommit.  thanks

Reply:
done